app/views/Export/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as ExportCodeInput } from './ExportCodeInput';
```

Comment:
Why not just export named consts from the sub modules if you're gonna do the dance here?

Reply:
Personally I like the internal / external abstraction better. 
IMO `ExportCodeInput` should export the view as the default without any context of how it'll be imported outside of the file to make the file more readable. 

I'm fine changing it to another export structure if we want though!

Reply:
I mostly care that you can type `ExportCodeInput` and VSCode autocomplete will find and import the correct file. If your current export pattern works, great.

Generally default exports only work 50% of the time.

I get the impression this index.js will have sufficient enough naming that it will work.

Reply:
Ah, gothcya. Yeah, this works for auto importiing